File: PP_02.01/DrivingSchoolWebAPI2/DrivingSchoolAPIModels/InstructorRating.cs
namespace DrivingSchoolAPIModels
{
    /// <summary>
    /// Инструктор с рейтингом
    /// </summary>
    public class InstructorRating : Instructor
    {
        public int NumberOfGrades { get; set; }
        public float Grade { get; set; }
    }
}

File: PDP/DrivingSchoolAppAndroidClient/app/src/main/java/com/example/drivingschoolappandroidclient/api/models/InstructorRating.kt
package com.example.drivingschoolappandroidclient.api.models

class InstructorRating(
    instructorId: Int,
    userId: String,
    user: ApplicationUser,
    var numberOfGrades: Int,
    var grade: Float
) : Instructor(instructorId, userId, user) {
}